src/UI-Constants/Constant.json

```diff
@@ -133,5 +133,54 @@
     "TIMEOUT_MEDIUM": 10000,
     "TIMEOUT_LONG": 30000,
     "TIMEOUT_XLONG": 60000
+  },
+  "TextConstants": {
+    "TITLE": "KUBAIR",
+    "BUTTON_TEXT": "Get Started",
+    "FOOTER_TEXT": "Your money is safe with us"
+  },
+  "Styles": {
+    "CONTAINER_FLEX": 1,
+    "IMAGE_MARGIN_TOP": 0.1,
+    "IMAGE_WIDTH": 0.44,
+    "IMAGE_HEIGHT": 0.44,
+    "TITLE_MARGIN_TOP": -0.1,
+    "TITLE_FONT_SIZE": 42,
+    "TITLE_FONT_FAMILY": "serif",
+    "TITLE_COLOR": "#3b4e48",
+    "TITLE_LETTER_SPACING": 8,
+    "BUTTON_BORDER_RADIUS": 120,
+    "BUTTON_TOP": 0.05,
+    "BUTTON_BACKGROUND_COLOR": "#000",
+    "BUTTON_PADDING_VERTICAL": 10,
+    "BUTTON_PADDING_HORIZONTAL": 50,
+    "BUTTON_TEXT_COLOR": "#e9e9d4",
+    "BUTTON_TEXT_FONT_WEIGHT": "bold",
+    "BUTTON_TEXT_ALIGN": "center",
+    "BUTTON_TEXT_WIDTH": 0.23,
+    "BUTTON_TEXT_HEIGHT": 0.029,
+    "BUTTON_TEXT_FONT_SIZE": 18,
+    "FOOTER_TEXT_BOTTOM": -0.353,
+    "FOOTER_TEXT_FONT_SIZE": 18,
+    "FOOTER_TEXT_ALIGN": "center",
+    "FOOTER_TEXT_FONT_WEIGHT": "bold",
+    "FOOTER_TEXT_COLOR": "#7a897f"
+  },
+  "AnimationDurations": {
```

Comment:
These can be clubbed with Animation key already used in this file.

Reply:
Animation styles modified and clubbed